include/nuttx/sched_note.h

```diff
@@ -336,7 +345,7 @@ struct note_string_s
 struct note_binary_s
 {
   struct note_common_s nbi_cmn; /* Common note parameters */
-  uint32_t nbi_module;          /* Module number */
+  uint8_t  nbi_module[4];       /* Module number */
```

Comment:
Based on provided use case description I do not see why we need change `uint32_t nbi_module;` to `uint8_t nbi_module[4];`?

Reply:
@pkarashchenko   variables keep the same style, only use uint8_t in struct of sched_note.h.

Reply:
Yes, but there is not benefit of doing it. Both `void sched_note_dump(uint32_t module, uint8_t event, FAR const void *buf, size_t len)` and `void sched_note_vbprintf(uint32_t module, uint8_t event, FAR const char *fmt, va_list va)` take `uint32_t module` as a parameter and it is naturally to store it into a same type placeholder. Even `NOTE_MODULE` macro creates an integer value. So why do we need this particular change?

Reply:
Another reason is to ensure that the saved data can be parsed by the machine with the different endian.

Reply:
@xiaoxiang781216 agree! That is 100% valid. This item can be resoled and only fix of `NOTE_MODULE` is remaining